source/_posts/2016-10-01-we-have-raspberry-image-now.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "We have a Raspberry Pi image now"
+description: "Release of the Rapsberry Pi Image for Home Assistant"
+date: 2016-10-01 01:00:00 -0400
+date_formatted: "October 1, 2016"
+author: Fredrik Lindqvist
+comments: true
+categories: Technology
+---
+
+## {% linkable_title We finally have a Raspberry Pi image! %}
```

Comment:
Didn't look as bad on a big screen but you are without a doubt correct. Working it into the text instead.